test/functional/ui/drop_spec.lua

```diff
@@ -0,0 +1,80 @@
+local helpers = require('test.functional.helpers')(after_each)
+local Screen = require('test.functional.ui.screen')
+local clear, feed, execute = helpers.clear, helpers.feed, helpers.execute
+
+describe("'drop'", function()
```

Comment:
Whoopsie, that's what I get from copying over other tests :) Will change, thanks!